### ISSUE TITLE
Clifford simulator states needs tests

### ISSUE THREAD

**Issue by dabacon** [quantumlib/Cirq#3021]
`clifford_tableau` and `stabilizer_state_ch_form` both have no tests.

**Comment by satw1knandala** [comment]
I'll take a crack at this.

**Comment by balopat** [comment]
@satw1knandala are you still up for helping out here? 
If yes, please consult with @smitsanghavi around this if you'd like to help out - he is working on major rework for the Clifford simulator, which includes covering some of the code paths here.

**Comment by satw1knandala** [comment]
Yes, I'm still down to help here. I will get into contact with him

**Comment by dstrain115** [comment]
The clifford simulator was completely written since this issue and now has tests.  See, for instance:

https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/sim/clifford/clifford_tableau_simulation_state_test.py
https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/sim/clifford/stabilizer_ch_form_simulation_state_test.py
